### ISSUE TITLE
[Migration] Perform guest kernel update and package installation inside guest and migrate

### ISSUE THREAD

**Issue by balamuruhans** [autotest/tp-libvirt#1643]
Call Trace inside VM was observed after migration when VM was upgrading the kernel and other packages during migration. Planning to automate the scenario for future verification.

**Comment by dzhengfy** [comment]
@balamuruhans , does the scenario mean to do 'yum upgrade' within guest during migration?

**Comment by balamuruhans** [comment]
@dzhengfy yes right, I observed this issue during early RHEL7.5-ALT phase.

**Comment by dzhengfy** [comment]
@balamuruhans I think you mean to run 'yum update' to update all needed packages for guest system. Right?

**Comment by balamuruhans** [comment]
@dzhengfy yes package install/update is performed inside guest system
